eZ/Publish/API/Repository/Tests/SearchServiceDeleteTranslationTest.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\API\Repository\Tests;
+
+use eZ\Publish\API\Repository\Tests\SetupFactory\LegacyElasticsearch;
+use eZ\Publish\API\Repository\Values\Content\Content;
+use eZ\Publish\API\Repository\Values\Content\Query;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
+use eZ\Publish\API\Repository\Values\Content\Search\SearchResult;
+
+/**
+ * Test case for delete content translation with the SearchService
+ *
+ * @see \eZ\Publish\API\Repository\SearchService
+ * @group integration
+ * @group search
+ */
+class SearchServiceDeleteTranslationTest extends BaseTest
```

Comment:
There's a lot of those already. Don't we have `SearchService`-is namespace already here or in `master`? Not asking to move everything, just this one (if indeed we have ns)